packages/hooks/src/hooks/useContract.ts

```diff
@@ -2,6 +2,16 @@ import React from 'react';
 import { Web3Context } from '../Provider';
 import { Contract } from 'ethers';
 
+/**
+ * @dev Hook to get the contract instance from ABI and contract address
+ * @param address contract address
+ * @param abi contract ABI
+ * @returns {
+ * contract: An instance of the current contract
+ * isReady: True when the contract is ready to use, false otherwise.
+ * }
+ */
+
 export function useContract(address: string, abi) {
```

Comment:
```suggestion
 * @returns {
 *   contract: An instance of the current contract
 *   isReady: True when the contract is ready to use, false otherwise.
 * }
 */

export function useContract(
  address: string, 
  abi
): [contract: any, isReady: boolean] {
```

It's hard to add a tuple return type in `jsdoc`. I think this is the best we can do.

![image](https://user-images.githubusercontent.com/1954752/148596950-29ccd888-44c1-4016-84fa-136a67f7b5a0.png)